### DISCUSSION TITLE
Are experimental features not subject to be removed?

### DISCUSSION THREAD

**Discussion by BradKwon** [firebase/flutterfire#10362] [Q&A] [answered]
Hi community,

Maybe this is a silly question but it would be great if someone can answer this.

I see there is an "experimental" feature introduced in `cloud_firestore' package.
For instance, `FEAT: experimental setIndexConfiguration() API `

The Firebase documentation says nothing about removal. However, I want to know if there is any possibility that this feature can be removed in the future. So I can check if I can use this feature in production.

Thank you in advance.

**Comment by ton-An** [comment, marked as answer]
From my understanding, experimental features can be altered or deleted at any point. But, judging from the feature's pull request #9928 and a related issue #9913 it will keep existing in some form or another.

**Comment by BradKwon** [comment]
@ton-An Thank you so much!
